distributed/http/worker/tests/test_worker_http.py

```diff
@@ -56,6 +58,27 @@ async def test_prometheus(c, s, a):
     await fetch_metrics(a.http_server.port, prefix="dask_worker_")
 
 
+@pytest.fixture
+def prometheus_not_available():
+    import sys
+
+    with mock.patch.dict("sys.modules", {"prometheus_client": None}):
+        del sys.modules["distributed.http.worker.prometheus"]
+        yield
+
+
+@gen_cluster(client=True, nthreads=[])
+async def test_metrics_when_prometheus_client_not_installed(
```

Comment:
This fails if you run it in isolation, because prometheus hasn't been imported:
```
================================================================================ ERRORS =================================================================================
__________________________________________________ ERROR at setup of test_metrics_when_prometheus_client_not_installed __________________________________________________

    @pytest.fixture
    def prometheus_not_available():
        import sys
    
        with mock.patch.dict("sys.modules", {"prometheus_client": None}):
>           del sys.modules["distributed.http.worker.prometheus"]
E           KeyError: 'distributed.http.worker.prometheus'

distributed/http/worker/tests/test_worker_http.py:66: KeyError
```